pkg/services/rpcsrv/server.go

```diff
@@ -2479,15 +2472,15 @@ func (s *Server) submitOracleResponse(ps params.Params) (any, *neorpc.Error) {
 	}
 	txSig, err := ps.Value(2).GetBytesBase64()
 	if err != nil {
-		return nil, neorpc.NewInvalidParamsError(fmt.Sprintf("tx signature is missing: %s", err))
+		return nil, neorpc.WrapErrorWithData(neorpc.ErrInvalidSignature, fmt.Sprintf("tx signature is missing: %s", err))
 	}
 	msgSig, err := ps.Value(3).GetBytesBase64()
 	if err != nil {
-		return nil, neorpc.NewInvalidParamsError(fmt.Sprintf("msg signature is missing: %s", err))
+		return nil, neorpc.WrapErrorWithData(neorpc.ErrInvalidSignature, fmt.Sprintf("msg signature is missing: %s", err))
 	}
 	data := broadcaster.GetMessage(pubBytes, uint64(reqID), txSig)
 	if !pub.Verify(msgSig, hash.Sha256(data).BytesBE()) {
-		return nil, neorpc.NewRPCError("Invalid request signature", "")
+		return nil, neorpc.ErrInvalidSignature
```

Comment:
And this one is `ErrInvalidSignature`, because the provided signature was checked against the public key.

Reply:
Thank you for the explanation